### ISSUE TITLE
Change how links are marked up

### ISSUE THREAD

**Issue by victorloux** [victorloux/tweening-icons#1]
There is a need to change how the links are marked up. Currently the links:

- do not work on mobile touch
- are not keyboard focusable
- are not announced correctly by screen readers (as links nor have any alt text)
- generally lack semantics (bad SEO)
- images do not render if JS is disabled 

Another solution should be implemented for progressive enhancement and make this proof of concept more usable in the wild.
